tests/cs/cs_test.cs

```diff
@@ -134,15 +134,18 @@ static void Main(string[] args)
             queryString = "drop database if exists foo; create database if not exists foo; use foo; drop table if exists t;create table t(ts timestamp, name varchar(20), mark nchar(20)); insert into t (ts, name, mark) values (now(), '测试', '人物')";
             execute_non_query(connString, queryString);
             execute_query(connString, "select * from foo.t");
+#if !FAKE_TAOS
```

Comment:
> why use FAKE_TAOS here?

Because the taos branch of odbc does not yet support the new date format yyyy-MM-ddTHH:mm:ss.fffZ, more specifically, in the function _stmt_param_copy_to_sql_timestamp, Currently only the format "%Y-%m-%d %H:%M:%S" can be processed.
In the future, we will enhance to support the new format